services/static-webserver/client/source/resource/osparc/new_studies.json

```
@@ -68,23 +68,23 @@
       "showDisabled": true,
       "expectedTemplateLabel": "mcTI Planning Tool",
       "title": "Multichannel TI",
-      "description": "Not available in TIP lite",
+      "description": "Not available in replace_me_product_name",
```

Minor:
If a token should be replaced, as a developer I find it more clear, if it looks something like below

```suggestion
      "description": "Not available in ${replace_me_product_name}",
```